### PR TITLE
Allow period in student usernames

### PR DESCRIPTION
**Pull request by LudvigHz**
NTNU has apparently started creating usernames with periods in them...

### CODE REVIEW

File: lego/apps/users/tests/test_student_confirmation_api.py [hunk 1]
```diff
@@ -181,6 +181,20 @@ def test_with_invalid_student_username(self, *args):
...
+    )
+    def test_with_student_username_with_period(self, *args):
+        AbakusGroup.objects.get(name="Users").add_user(
+            self.user_without_student_confirmation
+        )
+        self.client.force_authenticate(self.user_without_student_confirmation)
+        invalid_data = self._test_student_confirmation_data.copy()
+        invalid_data["student_username"] = "test.user"
+        response = self.client.post(_get_list_request_url(), invalid_data)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```
Comment (ivarnakken) [inline lego/apps/users/tests/test_student_confirmation_api.py:196]: Why are you expecting a bad request here? I though the point of the validator was to accept usernames with periods in them?

Comment (LudvigHz) [reply lego/apps/users/tests/test_student_confirmation_api.py:196]: whooops. I copied a test that tested for invalid usernames and forgot to modify the assert :upside_down_face:.
Thanks!

Comment (ivarnakken) [reply lego/apps/users/tests/test_student_confirmation_api.py:196]: ah okay! Remember to change the variable names as well, from "invalid" to "valid" 😅

**Review by ivarnakken — APPROVED** [review]
Looking good 💯 Just fix up isort

**Pull request merged**